internal/controllers/operator_controller_test.go

```diff
@@ -31,10 +32,13 @@ var _ = Describe("Operator Controller Test", func() {
 	)
 	BeforeEach(func() {
 		ctx = context.Background()
+		variableSource := olm.NewOLMVariableSource(cl)
+		solver := resolution.NewOperatorResolver(testEntitySource, variableSource)
```

Comment:
I prefer this for readability reasons (very subjective, obviously), but happy to change this. Will do that in the next update.

Reply:
Sorry! When I write "nit:" I don't mean to say you must change it - just a nit-pick that I wanted to note. I agree that this might be more readable in the code, but less readable in the diff. Perfectly OK for you to decide one way versus the other. No change needed :)